models/spring-ai-minimax/src/main/java/org/springframework/ai/minimax/api/MiniMaxApi.java

```diff
@@ -114,9 +114,15 @@ public MiniMaxApi(String baseUrl, String miniMaxToken, RestClient.Builder restCl
 	 * <a href="https://www.minimaxi.com/document/algorithm-concept">MiniMax Model</a>.
 	 */
 	public enum ChatModel implements ModelDescription {
-		ABAB_6_Chat("abab6-chat"),
+		ABAB_6_5_Chat("abab6.5-chat"),
+		ABAB_6_5_S_Chat("abab6.5s-chat"),
+		ABAB_6_5_T_Chat("abab6.5t-chat"),
+		ABAB_6_5_G_Chat("abab6.5g-chat"),
 		ABAB_5_5_Chat("abab5.5-chat"),
-		ABAB_5_5_S_Chat("abab5.5s-chat");
+		ABAB_5_5_S_Chat("abab5.5s-chat"),
+
+		@Deprecated(since = "1.0.0-M1", forRemoval = true) // Replaced by ABAB_6_5_S_Chat
```

Comment:
This should be 1.0.0-M2 as it gives users one release cycle to prepare for the removal.

Reply:
got it